roles/rhods_test_jupyterlab/tasks/main.yml

```diff
@@ -176,6 +176,27 @@
     oc delete ev -n {{ rhods_test_namespace }} --all > /dev/null
   failed_when: false
 
+- name: Update default notebook size
+  environment:
+    KUBECONFIG: '{{ sut_cluster_kubeconfig }}'
+    NOTEBOOK_RAM: '{{ rhods_test_jupyterlab_notebook_memory }}'
+    NOTEBOOK_CPU: '{{ rhods_test_jupyterlab_notebook_cpu }}'
+
+  command: |
+    oc get configmap -o yaml --namespace redhat-ods-applications odh-jupyterhub-global-profile > /tmp/odh-jh-gp.yaml
```

Comment:
when nothing confidential is handled, I prefer using ansible for this kind of things:
```
command: oc get configmap/odh-jupyterhub-global-profile -oyaml -n redhat-ods-applications 
register: jh_global_profile_cmd
```
Ansible is better for logging, as the content will be printed in the logs.
If the content is long (not the case here), then write into a file (that makes the logs harder to follow when troubleshooting)

(notice: no space between `-o ... $output`, and `--namespace --> -n`. For readability, I always use the short version of `-n`, but for everything else, I agree the long version is prefered)

Reply:
UPDATE: maybe it's simpler to use a file :#
then: use one ansible task per action, and use `{{ artifact_extra_logs_dir }}/jh_global_profile.json` as file destination